tests/lstsq/test_tikhonov.py

```diff
@@ -192,36 +193,36 @@ def test_process_regularizer(self, k=20, d=11, r=3):
         with pytest.raises(TypeError) as ex:
             solver._process_regularizer([1, 2, 3])
         assert ex.value.args[0] == \
-            "regularization hyperparameter λ must be a scalar"
+            "regularization hyperparameter lambda_par must be a scalar"
```

Comment:
This would change if `lambda_par` -> `regularizer`. In that case the message could just drop the argument name and say `"regularization hyperparameter must be a scalar"`